pkgs/applications/audio/nuclear/default.nix

```diff
@@ -0,0 +1,32 @@
+{ appimageTools, lib, fetchurl }:
+let
+  pname = "nuclear";
+  version = "0.6.6";
+  name = "${pname}-v${version}";
+
+  src = fetchurl {
+    url = "https://github.com/nukeop/nuclear/releases/download/v${version}/${name}.AppImage";
+    sha256 = "0c1335m76fv0wfbk07s8r6ln7zbmlqd66052gqfisakl8a1aafl6";
+  };
+
+  appimageContents = appimageTools.extractType2 { inherit name src; };
+in appimageTools.wrapType2 {
+  inherit name src;
+
+  extraInstallCommands = ''
+    mv $out/bin/${name} $out/bin/${pname}
+
+    install -m 444 -D ${appimageContents}/${pname}.desktop $out/share/applications/${pname}.desktop
+    substituteInPlace $out/share/applications/${pname}.desktop \
+      --replace 'Exec=AppRun' 'Exec=nuclear'
+    cp -r ${appimageContents}/usr/share/icons $out/share
+  '';
+
+  meta = with lib; {
+    description = "Streaming music player that finds free music for you";
+    homepage = "https://nuclear.js.org/";
+    license = licenses.agpl3Plus;
+    maintainers = [ maintainers.ivar ];
```

Comment:
```suggestion
    maintainers = with maintainers; [ ivar ];
```

Reply:
Why? The full maintainer list doesn't need to be in scope, and this looks a lot nicer IMO.

Reply:
It makes adding new maintainers easier.

Reply:
Well yes, but a second maintainer can also make that change when it is actually useful.

Reply:
You can also be nice and do it in the beginning. 🙃

Reply:
In my opinion this looks way nicer than your proposed solution, which as of now has no functional advantage. 

There is no guideline on this (as far as I'm aware), so I think this should be fine.